__init__.py

```diff
@@ -52,6 +53,7 @@
 #   Later weather: only the word "later" in the vocab file works all others
 #       invoke datetime skill
 
+FIFTEEN_MINUTES = 900
```

Comment:
Is this used anywhere or left in from dev?

Reply:
good catch.  I moved it to the Home Screen skill.  removing.